docs/Measuring_Resonances.md

```diff
@@ -42,14 +52,24 @@ Recommended connection scheme for I2C:
 | SDA | 03 | GPIO02 (SDA1) |
 | SCL | 05 | GPIO03 (SCL1) |
 
+![MPU-9250 connected to RPI](img/MPU9250-PI-fritzing.png)
 
-Fritzing wiring diagrams for some of the ADXL345 boards:
+Recommended connection scheme for I2C(i2c0a) on the RP2040:
 
-![ADXL345-Rpi](img/adxl345-fritzing.png)
+| MPU-9250 pin | RP2040 pin | RPi pin name |
+|:--:|:--:|:--:|
+| 3V3 (or VCC) | 39 | VSYS |
+| GND | 38 | Ground |
+| SDA | 01 | GP0 (I2C0 SDA) |
+| SCL | 02 | GP1 (I2C0 SCL) |
 
+![MPU-9250 connected to PICO](img/MPU9250-PICO-fritzing.png)
 
-Double-check your wiring before powering up the Raspberry Pi to prevent
-damaging it or the accelerometer.
+***I2C protocol is not reliable on long distance, consider hanging PICO
+within 5cm cable length away from the accelerometer!***
+
+***Double-check your wiring before powering up to prevent
```

Comment:
This comment now appears to be applicable only to MPU* accelerometers, and the users of ADXL345 may skip it and not read it. I would suggest to either duplicate it in ADXL345 section, or alternatively add it as a first paragraph to `### Wiring` section before going to any of the accelerometers.

Reply:
I could reliably run the MPU950 with a Cat5e cable of 50cm. My feeling is that the ADXL and MPU are equally demanding WRT cable quality.

Maybe worth to resurrect #5059

Reply:
It is probably a problem with my setup. I have all the wires basically in parallel vs twisted pairs, and I couldn't get anything to work  at around 30-10 cm and I had to shorten it to around 5 to work. Maybe it's a problem with pico?

Reply:
Tested both Pico and RPi. Could not see any differences between these two WRT cable.
Any regular Dupont jumper cable were giving me serious issues, even on the regular 10 to 20 cm lengths.
No problems since I switched to a shielded ethernet cable (Cat 5e) and [this wiring setup](https://github.com/Klipper3d/klipper/pull/5059#issuecomment-1001714082)

Reply:
I have updated the document with some more general suggestions. I'll still run some tests tomorrow and see if different combinations/lengths have different effects.

Reply:
I was actually referring only to the 'Double-check your wiring before powering up to prevent...' comment, and not to the I2C cable length limit. Alas, github review pulled a larger block than I intended, sorry for the confusion. So, I think that warning about double-checking the wiring makes sense to have for all wiring setups.

Reply:
Though I'm fine with the current version of the docs, modulo the fact that I haven't been using twisted pair and I cannot confirm or say contrary to the 'Suggested twisted pair order'(s). Also, the same as what Sineos has said, I'm not sure I'd go as far as to suggest replacing the pull up resistors. It is a fiddly and potentially dangerous process, and I suppose the people who have the equipment and skills to pull that are already aware of that possibility, WDYT?